_maps/ship_encounters/events/ruins/factory_impact.dmm

```diff
@@ -0,0 +1,415 @@
+"cx" = (/obj/effect/decal/cleanable/blood/footprints{dir = 8},/turf/open/floor/plasteel/grimy,/area/space)
+"cy" = (/obj/effect/decal/cleanable/blood/footprints{dir = 8},/turf/open/floor/plasteel,/area/space)
+"cz" = (/obj/structure/curtain,/obj/structure/door_assembly/door_assembly_glass,/turf/open/floor/plasteel,/area/space)
+"cA" = (/obj/item/weapon/shield/riot/buckler,/obj/item/weapon/twohanded/spear,/obj/effect/decal/remains/human,/turf/open/floor/carpet,/area/space)
+"cB" = (/obj/structure/table,/obj/item/weapon/cane,/turf/open/floor/carpet,/area/space)
+"cC" = (/obj/machinery/door/airlock/maintenance_hatch,/obj/machinery/door/firedoor/closed,/turf/open/floor/plasteel,/area/space)
+"cD" = (/obj/effect/decal/cleanable/blood/tracks{dir = 10},/turf/open/floor/plasteel,/area/space)
+"cE" = (/obj/effect/decal/cleanable/blood/tracks{dir = 8},/turf/open/floor/plasteel,/area/space)
+"cF" = (/obj/item/roller,/obj/effect/decal/cleanable/blood/tracks{dir = 8},/turf/open/floor/plasteel,/area/space)
+"cG" = (/turf/open/floor/wood,/area/space)
+"cH" = (/obj/structure/light_construct{dir = 4},/obj/structure/bed,/obj/item/weapon/bedsheet/black,/turf/open/floor/wood,/area/space)
+"cI" = (/obj/structure/barricade/wooden,/turf/open/floor/plasteel,/area/space)
+"cJ" = (/obj/machinery/door/airlock/hatch,/obj/machinery/door/poddoor,/turf/open/floor/plasteel,/area/space)
+"cK" = (/obj/item/weapon/shovel,/obj/effect/decal/cleanable/cobweb,/turf/open/floor/plasteel,/area/space)
+"cL" = (/obj/structure/mineral_door/wood,/turf/open/floor/plasteel,/area/space)
+"cM" = (/obj/effect/mob_spawn/human/alive{dir = 8},/turf/open/floor/plasteel,/area/space)
+"cN" = (/obj/effect/decal/remains/human,/turf/open/floor/plasteel,/area/space)
+"cO" = (/obj/structure/toilet{dir = 1},/turf/open/floor/plasteel,/area/space)
+"cP" = (/obj/structure/light_construct,/turf/open/floor/plasteel,/area/space)
+"cQ" = (/obj/structure/bedsheetbin,/obj/structure/table,/turf/open/floor/plasteel,/area/space)
+"cR" = (/obj/machinery/door/airlock/external,/turf/open/floor/plating,/area/space)
+"cS" = (/turf/closed/indestructible/rock,/area/space)
+"cT" = (/obj/structure/fans/tiny,/obj/machinery/door/airlock/external,/turf/open/floor/plating,/area/space)
+"cU" = (/turf/closed/wall/r_wall/rust,/area/no_entry)
+"cV" = (/obj/machinery/door/airlock/external,/turf/open/floor/engine,/area/no_entry)
+"cW" = (/turf/open/floor/engine,/area/no_entry)
+"cX" = (/obj/machinery/light/small{dir = 2},/turf/open/floor/engine,/area/no_entry)
+"cY" = (/obj/machinery/door/airlock/external,/obj/structure/fans/tiny,/obj/machinery/door/airlock/external,/obj/structure/fans/tiny,/turf/open/floor/engine,/area/no_entry)
+"cZ" = (/obj/structure/grille,/obj/structure/window/reinforced/fulltile,/obj/structure/fans/tiny,/obj/structure/grille,/obj/structure/window/reinforced/fulltile,/obj/structure/fans/tiny,/turf/open/floor/plating,/area/no_entry)
+"da" = (/obj/effect/landmark/dock_spawn/ruin,/turf/open/space,/area/space)

```

Comment:
Dock spawns are too close to the ruins, crashing the shuttles with no survivors.

Best to place them slightly north of ruins bc the dock landmark is on the south of shuttles

Reply:
not sure how to place them, could u take a peek?

Reply:
@TMTIME